frontend/deb/template_rules.go

```diff
@@ -0,0 +1,56 @@
+package deb
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"io"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/Azure/dalec"
+	"github.com/moby/buildkit/client/llb"
+)
+
+func Rules(spec *dalec.Spec, in llb.State, dir string) (llb.State, error) {
+	buf := bytes.NewBuffer(nil)
+
+	if dir == "" {
+		dir = "debian"
+	}
+
+	if err := WriteRules(spec, buf); err != nil {
+		return llb.Scratch(), err
+	}
+
+	return in.
+			File(llb.Mkdir(dir, 0o755, llb.WithParents(true))).
+			File(llb.Mkfile(filepath.Join(dir, "rules"), 0o770, buf.Bytes())),
+		nil
+}
+
+func WriteRules(spec *dalec.Spec, w io.Writer) error {
+	return rulesTmpl.Execute(w, &rulesWrapper{spec})
+}
+
+type rulesWrapper struct {
+	*dalec.Spec
+}
+
+func (w *rulesWrapper) Envs() fmt.Stringer {
+	b := &strings.Builder{}
+
+	for k, v := range w.Spec.Build.Env {
+		fmt.Fprintf(b, "export %s := %s\n", k, v)
```

Comment:
handle value with spaces
```suggestion
		fmt.Fprintf(b, "export %s := \"%s\"\n", k, v)
```

Reply:
This is `make` which would include literal quotes, which we don't want here.